TSE Review 01.

wrakc opened the pull request:
- Added linter files;
- Added gitignore files;
- Added the main files (index, and styles);
- Created the main index code and added the pictures;
- Styled the elements accordingly to the website cloned;
- Updated and fixed the readme file.
- Fixed linter issues using local linter

wrakc commented:
I am getting the following error at the GitHub check for linter errors:

_Error: ENOENT: no such file or directory, open '/src/src/stylelint.config.js'_

So I locally fixed the linter errors.

wrakc commented:
Thanks, I will make the changes as suggested!

wrakc commented:
I fixed the issues with the stickler clone and fixed the margins of the background image when appearing in high-resolution screens, also closed the pull request that I created and reopened this one.

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+/* responsiveness */
+
+.resize {
+  min-width: 1440px;
```
Causing horizontal overflow, change it to max-width instead to fix the issue

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+}
+
+.images img {
+  width: 358px;
```
last step to fix horizontal overflow is to set this width to 99%

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+/* Text article */
+
+.text {
+  display: flex;
```
CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      align-items: center;
   ```

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+  border-radius: 5px 0 0 5px;
+}
+
+.input-div {
```
CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      align-items: center;
   ```

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+
+/* footer */
+
+footer {
```
CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      justify-content: center;
   ```

rammazzoti2000 commented on assets/styles.css:
```diff
@@ -0,0 +1,189 @@
+  margin-top: 15px;
+}
+
+.useful_links ul {
```
CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      justify-content: center;
   ```

wrakc commented:
I fixed the horizontal overflow left according to the suggestion;
I fixed the CSS bad practices according to the inline comments.

wrakc commented:
- I fixed the horizontal overflow left according to the suggestion;
- I fixed the CSS bad practices according to the inline comments;
- Fixed some linter errors.
